web/css/source/extend/variables/_structure.less

```diff
@@ -65,6 +65,7 @@
 @page-wrapper__indent-left: @menu__width + @page-content__padding-horizontal;
 @page-wrapper__background-color: @neutral-95;
 
+@page-content__margin-horizontal: @content__indent;
```

Comment:
@GrimLink, thanks for PR.
One small comment: when is needed to add new variable (does not exist in parent theme) better to place it in 
`expand` folder, at next path: `./web/css/source/expand`

So in this case should be aligned:
1. `./web/css/source/expand/_variables.less` with new `@import './variables/_structure.less';`
2. created `./web/css/source/expand/variables/_structure.less` file
3. moved `@page-content__margin-horizontal: @content__indent;` into `_structure.less`